pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -1271,19 +1264,18 @@ func TestImportUserDefinedTypes(t *testing.T) {
 }
 
 const (
-	testPgdumpCreateCities = `CREATE TABLE cities (
+	testPgdumpCreateCities = `CREATE TABLE public.cities (
 	city VARCHAR(80) NOT NULL,
 	CONSTRAINT cities_pkey PRIMARY KEY (city ASC),
 	FAMILY "primary" (city)
 )`
-	testPgdumpCreateWeather = `CREATE TABLE weather (
+	testPgdumpCreateWeather = `CREATE TABLE public.weather (
 	city VARCHAR(80) NULL,
 	temp_lo INT8 NULL,
 	temp_hi INT8 NULL,
 	prcp FLOAT4 NULL,
 	date DATE NULL,
-	CONSTRAINT weather_city_fkey FOREIGN KEY (city) REFERENCES cities(city),
-	INDEX weather_auto_index_weather_city_fkey (city ASC),
```

Comment:
just curious, why have you dropped this INDEX? Same in one other spot above.

Reply:
This was changed in https://github.com/cockroachdb/cockroach/pull/50771